### PR TITLE
Add `PageHeader` Docs

### PR DESCRIPTION
**Pull request by jorytindall**
### :pushpin: Summary

This PR adds the documentation for the `PageHeader` feature branch (#1377).

### :link: External links

Jira ticket: [HDS-1893](https://hashicorp.atlassian.net/browse/HDS-1893)
Figma file: [PageHeader in Figma](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/branch/Urm6MreKfPoJMMW1sWTR08/HDS-Product---Components?type=design&node-id=34664%3A70976&t=ahkiFBhew9rT7u3B-1)

[HDS-1893]: https://hashicorp.atlassian.net/browse/HDS-1893?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by alex-ju — COMMENTED** [review]
Looks great, Jory! Nice to see it all together. Left a few suggestions.

Coming from a plain English/governmental environment I feel we might be overusing `pertain` in this document, but I might be biased 😁

File: website/docs/components/page-header/index.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: Page Header
+description: An informational element that displays the title of the page, relevant metadata, and page level actions.
```
[the remark this answers is not in the document]
Comment (jorytindall) [reply website/docs/components/page-header/index.md:3]: I had initially been using "page-level" but did some looking around and found conflicting information on this. I probably should have just gone with my gut 😄

File: website/docs/components/page-header/partials/guidelines/guidelines.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+
+### When not to use
+
+- To communicate page level information anywhere other than the top of the page.
+
+## Title
+
+Displays the title or topmost heading of the page. This should be limited to no more than four words and not exceed 50 characters.
+
+### Multiple `<h1>` elements
```
[the remark this answers is not in the document]
Comment (jorytindall) [reply website/docs/components/page-header/partials/guidelines/guidelines.md:17]: I'm torn on this topic as well, I think it's nice to use a heading and be able to link to it, but let me see if I can reword it

[the remark this answers is not in the document]
Comment (alex-ju) [reply website/docs/components/page-header/partials/guidelines/guidelines.md:17]: ah, good point to having a direct link 🤔

File: website/docs/components/page-header/partials/guidelines/guidelines.md [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+- To communicate metadata pertaining to the page; e.g., status, description, subtitle, etc.
+- To highlight page-level actions and functions.
+
+### When not to use
+
+- To communicate page-level information anywhere other than the top of the page.
+
+## Title
+
+Displays the title or topmost heading of the page. This should be limited to no more than four words and not exceed 50 characters.
```
Comment (apriofrost) [inline website/docs/components/page-header/partials/guidelines/guidelines.md:15]: Hi Jory, I think the "no more than four words" is hard to follow because there are page titles that are showing user-defined values. For example, the HCP Organization name, which is user-defined and is used on the organization dashboard page as the title. There is a character number limitation of 36, but no limitation on how many words it can contain (and it allows spaces). 
For the "50 characters" limitation, so far it seems fine to me - I know Terraform Cloud's org name has a 40 characters limitation, and most of the user-defined names on HCP have a 36 characters limitation. But my research is not exhaustive so there is a chance that the 50 limitation is not applicable. But I agree that this is a good rule of thumb in most of the scenarios.

Comment (apriofrost) [reply website/docs/components/page-header/partials/guidelines/guidelines.md:15]: I forgot to mention that the use case of having the org name be the page title is an ongoing ticket to be implemented here once the page header component is available: https://hashicorp.atlassian.net/browse/HCPF-639

File: website/docs/components/page-header/partials/guidelines/guidelines.md [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+
+### Small
+
+![Small Page Header variant](/assets/components/page-header/page-header-size-small.png =300x*)
+
+## Breadcrumb
+
+Displays a Helios [Breadcrumb](/components/breadcrumb) to communicate the application hierarchy and location to the user.
+
+![Page Header Breadcrumb](/assets/components/page-header/page-header-breadcrumb.png =700x*)
```
Comment (apriofrost) [inline website/docs/components/page-header/partials/guidelines/guidelines.md:39]: Suggest to update this example to match with the third screenshot in this ticket: https://hashicorp.atlassian.net/browse/HCPF-639

Reasons: 
- The first item for Organization uses an "org" icon. Suggested to be updated to the "dashboard" icon to match better with the type of the page (an org dashboard).
- The second item uses the outdated icon choice for "project" .  It has been updated to use the "doc" icon because we want to reserve the "folder" icon for the future resource hierarchy called "Folder", which sits in between Organization and Projects. 
- With multi-project, we introduced a project listing page which is currently missing in the breadcrumb

File: website/docs/components/page-header/partials/guidelines/guidelines.md [hunk 5]
```diff
@@ -0,0 +1,237 @@
...
+!!!
+
+## Responsiveness
+
+The Page Header component in Figma supports two sizes; `large` which accounts for the majority of desktop sizes and large tablets, and `small` which accounts for smaller tablet and mobile devices. The core difference between each variant is the vertical stacking of elements.
+
+The Ember component uses a variety of different methods to ensure fluidity and responsiveness:
+
+- By default, the component will fill the page layout it is used within. It does not have any padding or margin explicitly applied, allowing it to adapt to different layout and spacing methods.
+- The component has breakpoints by means of container queries (at `7680px` and one at `400px`) that account for the majority of content within the component and stack elements in a single column as the container shrinks.
```
Comment (apriofrost) [inline website/docs/components/page-header/partials/guidelines/guidelines.md:225]: Can we double-check the first breakpoint of 7680px?? I saw a different defition in the [Vercel deployment ](https://hds-showcase-g6krxiq38-hashicorp.vercel.app/components/page-header)that says:

> The component uses container queries to determine two breakpoints at 700px and 400px which stack the content in a single column.

File: website/public/assets/components/page-header/page-header-badges-do-metadata.png
Comment (apriofrost) [inline website/public/assets/components/page-header/page-header-badges-do-metadata.png:1]: A nitpick on the "Boundary cluster" related examples: the default name of a Boundary cluster is set as `boundary-cluster`, suggest we update the page title to match that.

Reason:
It would look more realistic and make the connection of a real example easier if we rename the page title to match the cluster-ID requirement for a Boundary cluster by removing the space between the words:

>  cluster_id: only ASCII letters, numbers, and hyphens are allowed.

File: website/docs/components/page-header/partials/guidelines/guidelines.md [hunk 6]
```diff
@@ -0,0 +1,237 @@
...
+- A flow that creates an object that is related to the page, e.g., creating a new cluster or adding a new user.
+- Launching or deploying a specific product instance.
+
+### Secondary button
+
+Use a `secondary` [Button](/components/button) in the Page Header when highlighting actions that are less important than the primary action, such as editing the content on a page, managing access, etc.
+
+### Tertiary button
+
+Use a `tertiary` [Button](/components/button) when highlighting low-priority actions like linking to external documentation.
```
Comment (apriofrost) [inline website/docs/components/page-header/partials/guidelines/guidelines.md:173]: Do we have an example in production that uses this pattern? 
Since the page header space is precious, only critical and page-level actions should be included in the actions area. I think the primary and secondary button option is enough, and there is no need to support a tertiary button.

A link to the documentation page can be included in the page description as an inline link (like the Packer example), or in the main page area in the empty state call-to-action area.  What do you think?

File: website/docs/components/page-header/partials/guidelines/guidelines.md [hunk 7]
```diff
@@ -0,0 +1,237 @@
...
+
+!!!
+
+## Actions
+
+Use [Buttons](/components/button) and [Dropdowns](/components/dropdown) in the Page Header to highlight functions and actions that impact the page as a whole. Examples of this include:
+
+- Creating new objects that are listed within the page
+- Surfacing management details like connecting to an API
+- Deleting or deactivating an object
```
Comment (apriofrost) [inline website/docs/components/page-header/partials/guidelines/guidelines.md:155]: For disruptive actions like this, I suggest we include it in a dropdown menu with a label like "Manage". If we put it on the top level as a primary or secondary button, it is prone to misclicks.

File: website/docs/components/page-header/partials/guidelines/guidelines.md [hunk 8]
```diff
@@ -0,0 +1,237 @@
...
+
+Use a primary [Dropdown](/components/dropdown) when the same action can be done in multiple different contexts. For example, taking the same action through a different interface or path; GUI, CLI, or API call.
+
+![Page Header primary dropdown](/assets/components/page-header/page-header-actions-do-primary-dropdown.png =853x*)
+
+!!!
+
+!!! Dont
+
+Don't use a `primary` [Dropdown](/components/dropdown) when combining different, unrelated actions. Instead, use the `secondary` color.
```
Comment (apriofrost) [inline website/docs/components/page-header/partials/guidelines/guidelines.md:198]: > Instead, use the `secondary` color.
Should this be "dropdown" instead of "color"?

File: website/public/assets/components/page-header/page-header-actions.png
Comment (apriofrost) [inline website/public/assets/components/page-header/page-header-actions.png:1]: The mockup here has the last item in the breadcrumb "Consul clusters" not matching the "current" style, also the '/' is unnecessary.

File: website/public/assets/components/page-header/page-header-actions-dont-primary-dropdown.png
Comment (apriofrost) [inline website/public/assets/components/page-header/page-header-actions-dont-primary-dropdown.png:1]: The mockup here has the last item in the breadcrumb "Consul clusters" not matching the "current" style, also the '/' is unnecessary.

File: website/public/assets/components/page-header/page-header-actions-do-primary-dropdown.png
Comment (apriofrost) [inline website/public/assets/components/page-header/page-header-actions-do-primary-dropdown.png:1]: The mockup here has the last item in the breadcrumb "Consul clusters" not matching the "current" style, also the '/' is unnecessary.

**Review by apriofrost — CHANGES_REQUESTED** [review]
Overall it looks fantastic! Thanks for the great work @jorytindall !
I left some comments for suggestions, questions, and nitpicks on the mockups.

File: website/docs/components/page-header/partials/guidelines/guidelines.md (hunk 6 above)
Comment (heatherlarsen) [reply website/docs/components/page-header/partials/guidelines/guidelines.md:173]: Can this be updated to use a more real-world example? I know there was one that came up in Nomad. My concern is that this example will lead to confusion since you've outlined elsewhere that external documentation links should go in the description.

File: website/docs/components/page-header/partials/specifications/anatomy.md [hunk 9]
```diff
@@ -0,0 +1,14 @@
+## Anatomy
+
+![Anatomy of pageHeader](/assets/components/page-header/page-header-anatomy.png)
```
Comment (heatherlarsen) [inline website/docs/components/page-header/partials/specifications/anatomy.md:3]: This needs a width associated with it.

File: website/docs/components/page-header/index.md [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+description: An informational element that displays the title of the page, relevant metadata, and page-level actions.
+caption: Displays the title of the page, metadata, and page-level actions.
+status: complete
+links:
+  figma: https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components?type=design&node-id=34664%3A70976&t=2f1KLqP6H7lnQQSz-1
+  github: https://github.com/hashicorp/design-system/tree/main/packages/components/addon/components/hds/page-header
+previewImage: assets/illustrations/components/page-header.jpg
+navigation:
+  keywords:
+    ['title', 'action bar', 'breadcrumb', 'landmark', 'banner']
```
Comment (heatherlarsen) [inline website/docs/components/page-header/index.md:12]:
```suggestion
    ['title', 'action bar', 'breadcrumb', 'landmark', 'banner', 'heading']
```

**Review by heatherlarsen — COMMENTED** [review]
Great work on these docs! Left a few minor notes.

File: website/docs/components/page-header/partials/guidelines/guidelines.md (hunk 5 above)
Comment (didoo) [reply website/docs/components/page-header/partials/guidelines/guidelines.md:225]: @apriofrost I've had the same doubt, and looking at the code seems the correct one is `768px` (so the `700px` is the one to change)

Comment (jorytindall) [reply website/docs/components/page-header/partials/guidelines/guidelines.md:225]: Good catch!

File: website/docs/components/page-header/partials/guidelines/guidelines.md (hunk 3 above)
Comment (jorytindall) [reply website/docs/components/page-header/partials/guidelines/guidelines.md:15]: Ok for now I'll leave the recommendation on the character limit, but remove the note on number of words. That's a good point on user generated titles.

File: website/docs/components/page-header/partials/guidelines/guidelines.md (hunk 4 above)
Comment (jorytindall) [reply website/docs/components/page-header/partials/guidelines/guidelines.md:39]: Great, thanks for the notes on outdated icons, I'll update! There will likely be other screens and examples that I'll have to clean up as well.

File: website/docs/components/page-header/partials/guidelines/guidelines.md (hunk 8 above)
Comment (jorytindall) [reply website/docs/components/page-header/partials/guidelines/guidelines.md:198]: Color refers to the property name, but I'll admit this doesn't read well. I'll clean it up a bit.

File: website/docs/components/page-header/partials/guidelines/guidelines.md [hunk 11]
```diff
@@ -0,0 +1,237 @@
...
+
+Refer to [responsiveness](#responsiveness) for more information on how the component adapts to different viewport and container sizes.
+
+### Large
+
+![Large Page Header variant](/assets/components/page-header/page-header-size-large.png =945x*)
+
+### Small
+
+![Small Page Header variant](/assets/components/page-header/page-header-size-small.png =300x*)
```
[the remark this answers is not in the document]
Comment (jorytindall) [reply website/docs/components/page-header/partials/guidelines/guidelines.md:33]: I've nested the section on responsiveness under the size since they are related and moved it to the end since it's less relevant than the other topics.

File: website/docs/components/page-header/partials/guidelines/guidelines.md [hunk 12]
```diff
@@ -0,0 +1,237 @@
...
+
+## Icon Tile
+
+Displays a Helios [IconTile](/components/icon-tile) as a visual indicator for the content of the page, object, or product branding.
+
+![Page Header Icon Tile](/assets/components/page-header/page-header-icon-tile.png =700x*)
+
+!!! Info
+
+Only the `medium` size of the [IconTile](/components/icon-tile) is supported in the Page Header.
```
[the remark this answers is not in the document]
Comment (jorytindall) [reply website/docs/components/page-header/partials/guidelines/guidelines.md:53]: I'm not sure how important this is to consumers, but I've added a note about not competing visually with the title which is the core reason.

File: website/docs/components/page-header/partials/guidelines/guidelines.md (hunk 6 above)
Comment (jorytindall) [reply website/docs/components/page-header/partials/guidelines/guidelines.md]: I've updated the example of this, the example from Nomad triggers a terminal instance that allows you interact with the job through a console.

Alternatively, since this isn't really a pattern we want to recommend broadly, we could exclude this from the docs. The support would still exist within the component, but would be "hidden" a bit. Downsides: might be confusing when to use a tertiary button (if at all).

File: website/docs/components/page-header/partials/guidelines/guidelines.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (heatherlarsen) [reply website/docs/components/page-header/partials/guidelines/guidelines.md]: Whenever we "restrict" something, it's nice to be as helpful in the guidance as possible.

**Review by heatherlarsen — APPROVED** [review]
Found one image that looks a little stretched. Looks good to go, though!

File: website/docs/components/page-header/partials/code/component-api.md [hunk 13]
```diff
@@ -0,0 +1,44 @@
...
+  <C.Property @name="<[PH].Title>" @type="yielded component">
+    Yields text and structured content within an `<h1>`
+    <br/><br/>
+    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+  </C.Property>
+  <C.Property @name="<[PH].Breadcrumb>" @type="yielded component">
+    A container that yields a Helios [Breadcrumb](/components/breadcrumb) component.
+  </C.Property>
+  <C.Property @name="<[PH].IconTile>">
+    Yields an `medium` IconTile in line with the page title. For details about its API check the [IconTile](/components/icon-tile?tab=code) component.
```
Comment (KristinLBradley) [inline website/docs/components/page-header/partials/code/component-api.md:17]:
```suggestion
    Yields a `medium` IconTile in line with the page title. For details about its API, check the [IconTile](/components/icon-tile?tab=code) component.
```

File: website/docs/components/page-header/partials/code/how-to-use.md [hunk 14]
```diff
@@ -0,0 +1,80 @@
+## How to use this component
+
+A simple invocation of the component requires a contextual `<PH.Title>` which yields it's content within an `<h1>`.
```
Comment (KristinLBradley) [inline website/docs/components/page-header/partials/code/how-to-use.md:3]:
```suggestion
A simple invocation of the component requires a contextual `<PH.Title>` which yields its content within an `<h1>`.
```

File: website/docs/components/page-header/partials/code/how-to-use.md [hunk 15]
```diff
@@ -0,0 +1,80 @@
...
+</Hds::PageHeader>
+```
+
+### Contextual components
+
+The Page Header uses a number of contextual components that either yield their content or render a specific Helios component.
+
+!!! Info
+
+The layout of this component is reponsive and adjusts based on the width of it's bounding container. The examples captured here may not be indicative of how the component will render within a specific application or layout.
```
Comment (KristinLBradley) [inline website/docs/components/page-header/partials/code/how-to-use.md:17]:
```suggestion
The layout of this component is responsive and adjusts based on the width of its bounding container. The examples captured here may not be indicative of how the component will render within a specific application or layout.
```

File: website/docs/components/page-header/partials/guidelines/guidelines.md [hunk 16]
```diff
@@ -0,0 +1,236 @@
...
+
+## Subtitle
+
+Displays a subtitle beneath the title to communicate metadata that does not change frequently or is not subject to changes within the application. Common examples of this are organization name, project name, unique IDs, resource names, etc.
+
+![Page Header Subtitle](/assets/components/page-header/page-header-subtitle.png =700x*)
+
+!!! Dont
+
+Don't use full sentences in the subtitle, use a [description](#description) instead.
```
Comment (KristinLBradley) [inline website/docs/components/page-header/partials/guidelines/guidelines.md:75]:
```suggestion
Don’t use full sentences in the subtitle, use a [description](#description) instead.
```

File: website/docs/components/page-header/partials/guidelines/guidelines.md [hunk 17]
```diff
@@ -0,0 +1,236 @@
...
+
+Use badges to express similar or related metadata pertaining to the page content.
+
+![Page Header Badges Metadata](/assets/components/page-header/page-header-badges-do-metadata.png =700x*)
+
+!!!
+
+!!! Dont
+
+Don't use more than three badges within the Page Header, instead explore ways to express additional metadata in the `customMetadata` area or move the content to the main page.
```
Comment (KristinLBradley) [inline website/docs/components/page-header/partials/guidelines/guidelines.md:61]:
```suggestion
Don’t use more than three badges within the Page Header, instead explore ways to express additional metadata in the `customMetadata` area or move the content to the main page.
```

File: website/docs/components/page-header/partials/guidelines/guidelines.md [hunk 18]
```diff
@@ -0,0 +1,236 @@
...
+
+![Page Header Full Sentence Subtitle](/assets/components/page-header/page-header-subtitle-dont-full-sentence.png =700x*)
+
+!!!
+
+## Description
+
+Displays a description beneath the title and subtitle to communicate more detailed information about the page, link out to external documentation and resources, and capture more generic information about the page. Not all pages need a description, especially if the title is explicit enough.
+
+We recommend limiting the Page Header description to 1-2 sentences. If a longer description is required, consider linking to external documentation or moving more detailed content to an interstitial component like a [Flyout](/components/flyout).
```
Comment (KristinLBradley) [inline website/docs/components/page-header/partials/guidelines/guidelines.md:85]:
```suggestion
We recommend limiting the Page Header description to 1–2 sentences. If a longer description is required, consider linking to external documentation or moving more detailed content to an interstitial component like a [Flyout](/components/flyout).
```

File: website/docs/components/page-header/partials/guidelines/guidelines.md [hunk 19]
```diff
@@ -0,0 +1,236 @@
...
+
+![Page Header Description](/assets/components/page-header/page-header-description.png =700x*)
+
+!!! Dont
+
+Don't include overly complex details, long-form content, or instructions in the description. This can add too much visual weight to the Page Header and can detract from the content on the main page.
```
Comment (KristinLBradley) [inline website/docs/components/page-header/partials/guidelines/guidelines.md:91]:
```suggestion
Don’t include overly complex details, long-form content, or instructions in the description. This can add too much visual weight to the Page Header and can detract from the content on the main page.
```

File: website/docs/components/page-header/partials/guidelines/guidelines.md [hunk 20]
```diff
@@ -0,0 +1,236 @@
...
+
+Instead, link to more complex content using an [Inline Link](/components/link/inline) with an optional icon to indicate the scope of the link (internal resource, external resource, etc).
+
+<Hds::PageHeader as |PH|>
+<PH.Title>HCP Packer Dashboard</PH.Title>
+<PH.IconTile @icon="packer-color" @color="packer" />
+<PH.Description>
+Channel created and managed automatically for you by Packer. For more information on how this channel is managed refer to the Packer <Hds::Link::Inline @icon="external-link" @href="#">documentation</Hds::Link::Inline>.
+</PH.Description>
+</Hds::PageHeader>
```
Comment (KristinLBradley) [inline website/docs/components/page-header/partials/guidelines/guidelines.md:107]: I recommend indenting nested elements for better code readability.

File: website/docs/components/page-header/partials/guidelines/guidelines.md [hunk 21]
```diff
@@ -0,0 +1,236 @@
...
+
+!!!
+
+## Actions
+
+Use [Buttons](/components/button) and [Dropdowns](/components/dropdown) in the Page Header to highlight functions and actions that impact the page as a whole. Examples of this include:
+
+- Creating new objects that are listed within the page
+- Surfacing management details like connecting to an API
+- Pausing or refreshing a service pertaining to the page
```
Comment (KristinLBradley) [inline website/docs/components/page-header/partials/guidelines/guidelines.md:143]: Should probably add periods after these list items for consistency with your other list items elsewhere

File: website/docs/components/page-header/partials/guidelines/guidelines.md [hunk 22]
```diff
@@ -0,0 +1,236 @@
...
+![Page Header Actions Tertiary Button](/assets/components/page-header/page-header-actions-tertiary-button.png =700x*)
+
+### Dropdowns
+
+Use a [Dropdown](/components/dropdown) in the Page Header to combine multiple secondary functions, tasks, or elements that assist the user in managing the information or objects on the page. Examples of this include:
+
+- Copying API credentials to manage the object remotely
+- Editing or changing settings
+- Deleting or deactivating an object
+- Linking to documentation
```
Comment (KristinLBradley) [inline website/docs/components/page-header/partials/guidelines/guidelines.md:171]: Should perhaps add periods for consistency

File: website/docs/components/page-header/partials/guidelines/guidelines.md [hunk 23]
```diff
@@ -0,0 +1,236 @@
...
+
+Use a primary [Dropdown](/components/dropdown) when the same action can be done in multiple different contexts. For example, taking the same action through a different interface or path; GUI, CLI, or API call.
+
+![Page Header primary dropdown](/assets/components/page-header/page-header-actions-do-primary-dropdown.png =853x*)
+
+!!!
+
+!!! Dont
+
+Don't use a `primary` color [Dropdown](/components/dropdown) when combining different, unrelated actions. Instead, use the `secondary` color.
```
Comment (KristinLBradley) [inline website/docs/components/page-header/partials/guidelines/guidelines.md:185]:
```suggestion
Don’t use a `primary` color [Dropdown](/components/dropdown) when combining different, unrelated actions. Instead, use the `secondary` color.
```

**Review by KristinLBradley — COMMENTED** [review]
Looks good. Added some suggested grammar and punctuation fixes.

File: website/docs/components/page-header/partials/guidelines/guidelines.md (hunk 20 above)
Comment (jorytindall) [reply website/docs/components/page-header/partials/guidelines/guidelines.md:107]: Ah, good catch. I think that was a prettier mistake.

File: website/docs/components/page-header/partials/guidelines/guidelines.md [hunk 24]
```diff
@@ -0,0 +1,238 @@
...
+- The component has breakpoints by means of container queries (at `768px` and one at `400px`) that account for the majority of content within the component and stack elements in a single column as the container shrinks.
+- Elements displayed inline with each other (e.g., title and badges) have `flex-wrap: wrap;` set to wrap elements when the available space is reduced.
+
+!!! Info
+
+Additional responsive characteristics are the responsibility of the consumer and dependent on the layout and spacing methods defined at the application level.
+
+!!!
+
+## Related
```
Comment (didoo) [inline website/docs/components/page-header/partials/guidelines/guidelines.md:235]: should we add "Button", "Breadcrumb", "Dropdown" too?

**Review by didoo — APPROVED** [review]
Pre-approving the changes, but I have added a bunch of comments expecially around the Component API documentation.

Maybe @alex-ju you want to do a final pass too?

File: website/docs/components/page-header/partials/code/component-api.md [hunk 25]
```diff
@@ -0,0 +1,44 @@
...
+### Contextual components
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="<[PH].Title>" @type="yielded component">
+    Yields text and structured content within an `<h1>`
+    <br/><br/>
+    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+  </C.Property>
+  <C.Property @name="<[PH].Breadcrumb>" @type="yielded component">
+    A container that yields a Helios [Breadcrumb](/components/breadcrumb) component.
```
[the remark this answers is not in the document]
Comment (jorytindall) [reply website/docs/components/page-header/partials/code/component-api.md:14]: Ah, thank you for catching that!

File: website/docs/components/page-header/partials/guidelines/guidelines.md (hunk 24 above)
Comment (jorytindall) [reply website/docs/components/page-header/partials/guidelines/guidelines.md:235]: If I remember correctly the blueprint inserts a comment recommending to only list two related components (I'm not sure why). In this case it's probably useful to capture the further nested components.

File: website/public/assets/components/page-header/page-header-badges-do-metadata.png
Comment (jorytindall) [reply website/public/assets/components/page-header/page-header-badges-do-metadata.png:1]: Updated these examples, thank you Liz!
